app/src/main/res/values/strings.xml

```diff
@@ -133,6 +133,7 @@
     <string name="dash" translatable="false"><xliff:g id="dash">—</xliff:g></string>
     <string name="development_settings" translatable="false">Development</string>
     <string name="general_settings">General</string>
+    <string name="ui_settings">User Interface</string>
```

Comment:
Please replace `ui_settings` with `user_interface_settings` so its easier for translators.